item/historical/scripts/DatasetMerger.ipynb

```diff
@@ -191,147 +185,141 @@
        "      <td>...</td>\n",
        "      <td>...</td>\n",
        "      <td>...</td>\n",
-       "      <td>...</td>\n",
        "    </tr>\n",
        "    <tr>\n",
-       "      <th>82553</th>\n",
```

Comment:
Similarly, if we insist on having the data displayed at the top of every notebook, we are going to have thousands of diff lines like this to wade through, every time the data changes even slightly.

I would suggest removing the output altogether. We can merge this PR without removing the output, but need to find a solution for the next PR.